amy/emails/views.py

```diff
@@ -136,8 +126,20 @@ def get_context_data(self, **kwargs):
         context["title"] = f'Scheduled email "{self.object.subject}"'
         return context
 
+    def form_valid(self, form: ScheduledEmailUpdateForm) -> HttpResponse:
+        result = super().form_valid(form)
 
-class ScheduledEmailRescheduleView(
+        ScheduledEmailLog.objects.create(
+            details="Scheduled email was changed.",
```

Comment:
Could you include the user that made the change here?

This would be useful for reschedule/cancel log entries too.

Reply:
It's a good idea to add author to these log items. I have created a ticket for that: https://github.com/carpentries/amy/issues/2484 (it would require changes in multiple places so I didn't want to work on it in this PR).